led_map:
  red: 0
  blue: 1
  green: 2
  "off": 3

mosfet_device_map:
  # These devices are assigned
  # to as specific MOSFET device
  arm_laser: 1
  heater_0: 7
  heater_2: 9
  uv_led_carousel: 4
  uv_led_end_effector: 1
  white_led: 5

science_serial:
  baudrate: 38400
  timeout: 0
  port: "/dev/ttyTHS0"

science_info:
  max_error_count: 20
  sleep: 0.01
  uart_transmit_msg_len: 30

